### ISSUE TITLE
Game crash when pasting C# in Micro Processor

### ISSUE THREAD

**Issue by Odepax** [Anuken/Mindustry#5344]
**Platform**: Windows 10

**Build**: release build 126.2

**Issue**: The game crashes when pasting random shit in the Edit Processor screen. #5176 indicates it's fixed in 7.0, but is there even a 7.0?

In my case, I had some C# in the clipboard:

```cs
sealed class Symbol<T> {}

sealed class SymbolDictionary {
	readonly Dictionary<object, object> Storage = new();

	public SymbolDictionary Set<T>(Symbol<T> key, T value) {
		Storage[key] = value;

		return this;
	}

	public bool TryGet<T>(Symbol<T> key, [NotNullWhen(true)] out T? @out) {
		if (Storage.TryGetValue(key, out var value)) {
			@out = (T) value;
			return true;
		}

		@out = default;
		return false;
	}

	public T Get<T>(Symbol<T> key) => Storage.TryGetValue(key, out var value) ? (T) value : throw new Exception();
}
```

The C# itself isn't interesting, but does its job triggering the bug. Interestingly, the following JS code resulted in Invalid blocks, but no crash:

```js
((invertImages, invertCode, applyBlackBackground) => {
	document.body.style.filter = "invert()"
	document.body.style.background = applyBlackBackground ? "black" : "white"
	Array.from(document.querySelectorAll("pre")).map(pre => pre.style.filter = invertCode ? "invert()" : null)
	Array.from(document.querySelectorAll("img")).map(img => img.style.filter = invertImages ? "invert()" : null)
})(/* Invert Image | Invert Code | Dark BG */ 1, 0, 1)
```

**Steps to reproduce**:
1. Build a Micro Processor
2. Click your wonderful Micro Processor
3. Enter the config screen with the edit button that appears
4. Click Edit at the bottom
5. Ensure you have some random shit in your clipboard (see above)
6. Click Import from Clipboard

**Link(s) to mod(s) used**: No mod

**Save file**: Attached a `.msav`, but I had to rename it and append `.zip` to please GitHub...

[The-micro-processor-bug.msav.zip](https://github.com/Anuken/Mindustry/files/6594243/The-micro-processor-bug.msav.zip)

**(Crash) logs**:  Quite unsure about it.

[The-micro-processor-crash.txt](https://github.com/Anuken/Mindustry/files/6594293/The-micro-processor-crash.txt)

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [x] **I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.**
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by BlueWolf3682** [comment]
Why are you even pasting C# into Java based stuff?!

**Comment by Odepax** [comment]
> Wow! Processors! Looks cool!
> Edit!
> Edit I said!
> I can paste stuff? How so?
> Clipboard? Heck, I don't even know what I have in the clipboard...
> What can go wrong!

And this is how...

**Comment by Anuken** [comment]
> [...] indicates it's fixed in 7.0, but is there even a 7.0?

Yes.

**Comment by oinkymoo** [comment]
> indicates it's fixed in 7.0, but is there even a 7.0?

fixed in 7.0 means that it has been fixed, but the fix is not yet available to the public (i mean, it could be in bleeding-edge now) but will be available when the update is released
